README.md

```diff
@@ -19,6 +19,7 @@
 | LinqKit.Microsoft.EntityFrameworkCore 6 | [![Nuget](https://img.shields.io/badge/nuget-v6.1.5-blue) ![Nuget](https://img.shields.io/nuget/dt/LinqKit.Microsoft.EntityFrameworkCore)](https://www.nuget.org/packages/LinqKit.Microsoft.EntityFrameworkCore/6.1.0) | Microsoft.EntityFrameworkCore<ul><li>≥ 6.0.0</li></ul> | <ul><li>net6.0</li></ul>|
 | LinqKit.Microsoft.EntityFrameworkCore 7 | [![Nuget](https://img.shields.io/badge/nuget-v7.1.5-blue) ![Nuget](https://img.shields.io/nuget/dt/LinqKit.Microsoft.EntityFrameworkCore)](https://www.nuget.org/packages/LinqKit.Microsoft.EntityFrameworkCore/7.0.0) | Microsoft.EntityFrameworkCore<ul><li>≥ 7.0.0</li></ul> | <ul><li>net6.0</li><li>net7.0</li></ul>|
 | LinqKit.Microsoft.EntityFrameworkCore 8 | [![Nuget](https://img.shields.io/badge/nuget-v8.1.5-blue) ![Nuget](https://img.shields.io/nuget/dt/LinqKit.Microsoft.EntityFrameworkCore)](https://www.nuget.org/packages/LinqKit.Microsoft.EntityFrameworkCore/8.0.0) | Microsoft.EntityFrameworkCore<ul><li>≥ 8.0.0</li></ul> | <ul><li>net8.0</li></ul>|
+| LinqKit.Microsoft.EntityFrameworkCore 9 | [![Nuget](https://img.shields.io/badge/nuget-v8.1.5-blue) ![Nuget](https://img.shields.io/nuget/dt/LinqKit.Microsoft.EntityFrameworkCore)](https://www.nuget.org/packages/LinqKit.Microsoft.EntityFrameworkCore/9.0.0) | Microsoft.EntityFrameworkCore<ul><li>≥ 9.0.0</li></ul> | <ul><li>net9.0</li></ul>|

```

Comment:
badge version probably needs to be v9.1.5 here.

Reply:
Thx, for the input, I fixed it.